generators/app/templates/gulp/tasks/serve.js

```diff
@@ -33,9 +33,11 @@ var serveTask = function (gulp, plugins, config, helpers, generator_config) {
     gulp.watch(path.join(config.src.base, config.src.styles), ['styles-watch']);
     <% if(projectType == 'fe') { %>
     gulp.watch(config.src.templatesWatch, ['templates-watch']);
+    gulp.watch(path.join(config.src.base, 'scripts/vendor.json'), ['vendor-watch', 'templates-watch']);
     <% } %>
     gulp.watch(path.join(config.src.base, config.src.assets), ['assets-watch']);
     <% if(projectType == 'wp-with-fe') { %>
+    gulp.watch(path.join(config.src.base, 'scripts/vendor.json'), ['vendor-watch']);
```

Comment:
I'm probably nitpicking but can we keep the same order of templatesWatch and vendor-watch here and above?